app/models/ability.rb

```diff
@@ -24,6 +24,7 @@ def initialize(user)
 
     can :read, User, sequence_num: user.sequence_num, company_id: user.company_id
```

Comment:
Remove now that you have added new ability for read

Reply:
done